index.html

```diff
@@ -48,6 +49,172 @@ <h4 class="social-title">LET'S CONNECT</h4>
                 <a href="#"> <img src="./images/social/Hand2.svg" alt="Social media"/></a>
             </li>
         </ul>    
+    </section>
+                                        <!-- page 2 -->
+                                    <!-- Section 2 : Works -->
+    <section class="work">
+        <ul>
+        <!-- First Card -->
+            <li class="card-container1">
+                <img src="./images/Snapshoot1.png" class="card-container-img1" alt="Snapshoot1">
+
+                <h2 class="title-project1">Tonic</h2>  
+
+                <div class="card-content1">
+                    <ul class="type1">
+                        <p class="t1">CANOPY</p>
+                        <p class="t2">
+                            <img src="./images/Circle.png"/> 
+                            Back End Dev
+                        </p>
+                        <p class="t3"> <img src="./images/Circle.png"/> 2015</p>
+                    </ul>                     
+                </div> 
+                    
+                    <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    
+                    <ul class="tech1">
+                        <li class="language1">html</li>
+                        <li class="language2"> css</li>
+                        <li class="language3">javascript</li>
+                    </ul>
+
+                    <button type="button" class="see-proj1">See Project</button> 
+                </div>
+            </li>
+            <!-- Second Card -->
+            <li class="card-container2">
+                <img src="./images/Snapshoot2.png" class="card-container-img2" alt="Snapshoot2">
+
+                <h2 class="title-project2">Multi-Post <br> Stories</h2>
+                
+                <div class="card-content2">
+                    <ul class="type2">
+                       <p class="t1">CANOPY</p>
+                       <p class="t2"><img src="./images/Circle.png"/> Back End Dev</p>
+                       <p class="t3"><img src="./images/Circle.png"/> 2015</p>
+                    </ul>
```

Comment:
`ul` tags must contain `li` tags in them. Which then could have contained `p` tags or `a tags, whatever we need. so that browser could read it as an unordered list of tags for better browser site contact detection.